sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -33,6 +36,11 @@ public class HealthcareOperation : Operation<RecognizeHealthcareEntitiesResultCo
         /// </summary>
         public override string Id { get; }
 
+        /// <summary>
+        /// Gets an Status representing the operation.
```

Comment:
Consider something like
```suggestion
        /// Gets the status of the operation
```